examples/contract-sdk/c10l-hello-world/Cargo.toml

```diff
@@ -1,16 +1,16 @@
 [package]
 name = "c10l-hello-world"
 version = "0.0.0"
-edition = "2018"
+edition = "2021"
 license = "Apache-2.0"
 
 [lib]
 crate-type = ["cdylib"]
 
 [dependencies]
 cbor = { version = "0.5.1", package = "oasis-cbor" }
-oasis-contract-sdk = { git = "https://github.com/oasisprotocol/oasis-sdk" }
-oasis-contract-sdk-storage = { git = "https://github.com/oasisprotocol/oasis-sdk" }
+oasis-contract-sdk = { git = "https://github.com/oasisprotocol/oasis-sdk", tag = "contract-sdk/v0.3.0" }
```

Comment:
did these used to use main branch?

Reply:
Yeah, because the other tags were very old. We are now more stable and will tag releases more regularly :-)